gatsby-node.js

```diff
@@ -42,6 +42,107 @@ const getMessages = (path, language) => {
   }
 }
 
+/**
+ * Markdown isOutdated check
+ * Parse headers in markdown file (both translated and english) and compare their info structure.
```

Comment:
```suggestion
 * Parse header ids in markdown file (both translated and english) and compare their info structure.
```
Let's make it clear we're talking about the "header id" (open to another name) - ultimately it's the [URI fragment](https://en.wikipedia.org/wiki/URI_fragment) of the header. The reason we do this is that the header text will differ from language to language & we want to keep links consistent.